app/models/article.rb

```diff
@@ -18,6 +18,7 @@ class Article < ApplicationRecord
   belongs_to :job_opportunity, optional: true
   belongs_to :organization, optional: true
   belongs_to :collection, optional: true, touch: true
+  
```

Comment:
Did something change on the article model? It seems like maybe this was just an accidental enter.